### DISCUSSION TITLE
connecting nocodb to a WordPress database (MySQL) [Apr 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1994] [Discord Threads]
> 🙋‍♂️ **robertandrews**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### Basically you just need to know the DB connection string for your database  
#  
> 👨‍🚀 **wingkwong**  
#### if u r using WordPress, then the connection string can be found in wp-config.php  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969308749460279366)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
